configure.ac

```diff
@@ -1451,6 +1457,26 @@ else
   fi
 fi
 
+dnl enable external signer support
```

Comment:
I should also rename this to `--disable-external-signer` given the default is enable when a recent enough Boost version is found. I'll update the version info as well, although I believe it will throw if you try `--enable-external-signer` on a old version of Boost.

Reply:
Added, and clarified the commit to indicate external signer is an upcoming feature.